sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +557,141 @@ public async Task ValidateMergeCertificate()
             CollectionAssert.AreEqual(mergedServerCertificate.Cer, completedServerCertificate.Cer);
         }
 
-        // Backup Restore
-        // GetCertificates
-        // GetCertificatsIncludePending
-        // GetCertificateVersions
-        // GetDeletedCertificates
-        // GetUpdatePolicy
-        // IssuerCrud
-        // ContactsCrud
+        [Test]
+        public async Task VerifyGetIssuer()
+        {
+            string issuerName = WellKnownIssuerNames.Self;
+
+            // Will update the name once get correct provider name.
+            string providerName = "testProvider";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+            Response<CertificateIssuer> getIssuerResponse = await Client.GetIssuerAsync(issuerName);
+
+            CertificateIssuer getIssuer = getIssuerResponse.Value;
+
+            Assert.NotNull(getIssuer);
+            Assert.NotNull(getIssuer.Id);
+            Assert.AreEqual(issuerName, getIssuer.Name);
+        }
+
+        [Test]
+        public async Task VerifyUpdateIssuer()
+        {
+            string issuerName = WellKnownIssuerNames.Self;
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+
+            string updateIssuerName = "updateIssuer";
+            IssuerProperties issuerProperties = new IssuerProperties(updateIssuerName);
+            Assert.NotNull(issuerProperties.Name);
+            Assert.NotNull(issuerProperties.Id);
+
+            issuer = new CertificateIssuer(issuerProperties);
+
+            Response<CertificateIssuer> updateIssuerResponse = await Client.UpdateIssuerAsync(issuer);
+
+            CertificateIssuer updateIssuer = updateIssuerResponse.Value;
+
+            Assert.NotNull(updateIssuer);
+            Assert.NotNull(updateIssuer.UpdatedOn);
+            Assert.AreEqual(updateIssuerName, updateIssuer.Name);
+        }
+
+        [Test]
+        public async Task VerifyGetContacts()
+        {
+            var contact1 = new CertificateContact
+            {
+                Name = Recording.GenerateId(),
+                Phone = Recording.GenerateId(),
+                Email = "test@microsoft.com"
+            };
+
+            var contact2 = new CertificateContact
+            {
+                Name = Recording.GenerateId(),
+                Phone = Recording.GenerateId(),
+                Email = "test1@microsoft.com"
+            };
+
+            var contacts = new List<CertificateContact>
+            {
+                    contact1,
+                    contact2
+            };
+
+            RegisterForCleanUpContacts(contacts);
+
+            await Client.SetContactsAsync(contacts);
```

Comment:
This test failed beacuse an exception throw when set contacts. It seems like a bug from the exception message.  Could you please help us confirm this.
Exception: 'Service request failed.Status: 404 (Not Found).... <The resource you are looking for might have been removed, had its name changed, or is temporarily unavailable.</h3>

Reply:
Not sure. You should enable tracing (see our troubleshooting steps in the readme) and verify what is getting sent. You might also try using a valid (any country) phone number. I doubt it's verified, but really not sure - perhaps a generated ID is too short or something.

Reply:
Checked the request URI send by code found it's difference with REST API provide.

Code send request : {vaultBaseUrl}/**contacts**?api-version=7.0
[REST API provide](https://docs.microsoft.com/en-us/rest/api/keyvault/setcertificatecontacts/setcertificatecontacts
): {vaultBaseUrl}/**certificates/contacts**?api-version=7.0

Reply:
Good catch. Please fix the runtime code and update the certificates CHANGELOG.md file.

Reply:
Please make the fix and update the CHANGELOG.md.

Reply:
fixed.